src/lib/swapper/swappers/ThorchainSwapper/utils/makeSwapMemo/makeSwapMemo.ts

```diff
@@ -46,6 +43,7 @@ type MakeSwapMemoArgs = {
   buyAssetId: string
   destinationAddress: string
   limit: string
+  affiliateBps?: string
```

Comment:
does this need to be optional now that it's defined in the selector?

Reply:
Yeah, looks like we getOrDefault (default being '0' as a string, i.e a truthy value) affiliate bps, so making this required sounds more accurate

Reply:
Indeed! Make required in https://github.com/shapeshift/web/pull/4353/commits/47b3c94b52c5a0bb478160ced5b91a2c9b764122.